lib/chunk.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Chunk
+ * @constructor
+ * @memberOf BlockMap
+ * @param {Buffer} buffer
+ * @param {Number} position
+ * @returns {Chunk}
+ */
+function Chunk( buffer, position ) {
+
+  if( !(this instanceof Chunk) ) {
```

Comment:
Can't we just use `new` when we create objects?

Reply:
We do, but since others might consume this module (who might not), and this is the convention of guarding against global context – this is checked here.